Source/WebInspectorUI/UserInterface/Views/LocalResourceOverridePopover.css

```diff
@@ -66,7 +66,6 @@
     appearance: unset;
     border: 1px solid hsl(0, 0%, 78%);
     background: var(--background-color-content);
-    border-color: var(--text-color-quaternary);
```

Comment:
i think we actually want to override the above to be
```css
border: 1px solid var(--text-color-quaternary);
```
since we want to keep the same behavior as before (i.e. the last override wins)